automated install/basic-install.sh

```diff
@@ -1816,6 +1820,7 @@ finalExports() {
     echo "INSTALL_WEB_SERVER=${INSTALL_WEB_SERVER}"
     echo "INSTALL_WEB_INTERFACE=${INSTALL_WEB_INTERFACE}"
     echo "LIGHTTPD_ENABLED=${LIGHTTPD_ENABLED}"
+    echo "CACHE_SIZE=${CACHE_SIZE}"
```

Comment:
Does this value need to be echo'd back?  The way this is currently implemented, it's appending these settings to setupVars.conf instead of replacing them.  This requires the setting be unnecessarily deleted first.  
The cache size can be confirmed in the settings in the GUI if needed.

Reply:
Seems logical to do so as we output the rest of the values.